website/pages/blog/announcing-the-vercel-source-plugin.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Announcing The Vercel Source Plugin
+tag: announcement
+date: 2022/12/14
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Vercel source plugin.
+author: kemal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Gandi source plugin for CloudQuery.
```

Comment:
```suggestion
CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Vercel source plugin for CloudQuery.
```

Also I think we could drop the SQL bit and use `CloudQuery is an open source high performance data integration platform designed for security and infrastructure teams.` from [our main page](https://www.cloudquery.io/) as we have non SQL destinations.

Reply:
@disq can you update that to match our tagline ?